src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -1180,6 +1180,34 @@ private void UpdateDefaultTargets(ProjectRootElement currentProjectOrImport)
             }
         }
 
+        private string ResolveSdk(string sdkSpec, IElementLocation location)
+        {
+            string sdkName;
+            string sdkVersion;
+
+            string[] elements = sdkSpec.Split(new char[] { '/' }, 2);
+
+            //  TODO: Add error message text for InvalidSdkAttribute
+            ProjectErrorUtilities.VerifyThrowInvalidProject(elements.Length == 2, location, "InvalidSdkAttribute");
+
+            sdkName = elements[0];
+            sdkVersion = elements[1];
+
+            var sdksPath =_data.GetProperty(ReservedPropertyNames.msbuildSdksPath);
+            //  TODO: Add error message text.  Also verify whether this should actually be an "Invalid Project" exception - probably not
+            ProjectErrorUtilities.VerifyThrowInvalidProject(sdksPath != null, location, "PropertyMustBeSpecified", ReservedPropertyNames.msbuildSdksPath);
+
+            var sdkPath = Path.Combine(sdksPath.EvaluatedValue, sdkName, sdkVersion, "Sdk");
```

Comment:
@dsplaisted @rainersigwald 
Rainer and I discussed this earlier. This isn't an sdkVersion so it shouldn't be included in the path. Future code should probably have a check here `if (sdkVersion < realSdkVersionFromResolutionCode) { error() }`. But without the real SDK version we might as well just drop the `sdkVersion` entirely (ignore everything after the `/`).